src/main/java/command/CriteriaCommand.java

```diff
@@ -0,0 +1,31 @@
+package command;
+
+import allocator.Allocator;
+import exceptions.SEPException;
+import studentlist.StudentList;
+import ui.UI;
+
+public class CriteriaCommand extends Command {
+    private String input;
+    private UI ui;
+    private Allocator allocator;
+
+    public CriteriaCommand(StudentList studentList,String input, UI ui) {

```

Comment:
Similarly, a space after `StudentList studentList,` and before `String input`

Reply:
Good catch!